io.bytes.per.checksum	-392837573	ERROR	Task: attempt_1701312551309_0336_r_000000_0 - exited : org.apache.hadoop.HadoopIllegalArgumentException: Invalid checksum type: userOpt=null, default=CRC32C:-392837573, effective=null		30